packages/react-native-aztec/android/settings.gradle

```diff
@@ -1,2 +1,2 @@
-rootProject.name = 'react-native-aztec'
+rootProject.name = '@wordpress/react-native-aztec'
```

Comment:
This is not actually necessary for this PR, we will need to update the android and iOS build later anyway.
It was probably updated from the search and replace I did, I don't mind reverting if necessary